components/LessonCard.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+
+import '../scss/lessonCard.scss'
+
+type Props = {
+  coverImg: string
+  title: string
+  challengeCount: number
+  lessonCount: number
+  hourCount: number
+  description: string
+}
+
+const LessonCard: React.FC<Props> = props => {
+  return (
+    <>
+      <div className="card shadow-sm mt-5">
+        <div className="card-body p-1">
+          <div className="row no-gutters">
+            <div className="cover-img col-6 col-sm-6 col-md-3 col-lg-3 col-xl-3 pl-1 pr-4 py-2 w-25 mx-auto">
```

Comment:
remove `col-lg-3` and `col-xl-3` because `col-md-3` takes care of >= 768px. Source: https://getbootstrap.com/docs/4.4/layout/grid/